### PR TITLE
feat: Update readme

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

Update the readme about the installation.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,17 +19,45 @@
...
 
-```bash
-go install github.com/caicloud/ormb
+You can install the pre-compiled binary, or compile from source.
+
+### Install the pre-compiled binary
+
+Download the pre-compiled binaries from [the releases](https://github.com/caicloud/ormb/releases) page and copy to the desired location.
+
+### Compile from source
```
Comment (simon-cj) [inline README.md:34]: Do it need to provide docker image in the future.

Comment (gaocegege) [reply README.md:34]: Oh yeah, let's open an issue for it.

Comment (gaocegege) [reply README.md:34]: We should not use images in caicloud internal registry.

Comment (simon-cj) [reply README.md:34]: emmm, SGTM

Comment (gaocegege) [reply README.md:34]: https://github.com/caicloud/ormb/issues/54

**Pull request merged**